src/main/webapp/organization_search_area.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+document.addEventListener("DOMContentLoaded", async function() {
+  // This checks that requester User has valid credentials to edit/delete/view ALL organizations. (by checking userID)
+  let isMaintainer = false;
+  let forOrganizationsPage = false;
+
+  if (document.getElementById('search-area')) {
```

Comment:
This is looking great! Question: why do we have a if/else here?

Reply:
It looks like it is to decide if we are creating the search area for the organizations tab. I think it is to solve the issue of if we are showing the edit/delete buttons or not

Reply:
IMO we can go one further and a different JS for organizations.html (organizations_page.js) and index.html (landing_page.js)

That gets rid of these if statements, breaking the cycle of the the organization search area know about the page it is being rendered on. I'm only proposing moving/changing lines 15-28 here.

Feel free to do that in another PR though!